config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/ServiceAndPolicyLoaderUtil.java

```diff
@@ -55,8 +67,8 @@ public static String handleDuplicatePathName(Map<String, ? extends Folderable> b
         int duplicateCounter = 2;
         String basePath = entity.getPath();
         String clonePath = basePath;
-
-        if (Boolean.getBoolean(HANDLE_DUPLICATE_NAMES)) {
+        final String handleDuplicates = System.getProperty(HANDLE_DUPLICATE_NAMES);
```

Comment:
Can you please move this (reading the property) to a separate method as you did for others.